source/blog/2015-09-22-rspec-compsable-matchers.md

```diff
@@ -0,0 +1,218 @@
+---
+title: How the RSpec 3.0 composable matchers work
+author: Sam Phippen
+---
+
+##Warmup: the `===` protocol
+
+In Ruby, a number of objects implement the `===` protocol. Some of the objects
+that implement this protocol include:
+
+* all class objects
+* lambdas
+* regexes
+* ranges
+
+Conceptually, this protocol represents a "match", but can be thought of as
+categorising instances of objects. That is, the object that implements the
+`===` declares whether or not it belongs in the same category as the other
```

Comment:
> the object that implements the `===` declares...

This should be "the object that implements the `===` method declares..." or "the object that implements `===` declares..."

Also, IMO, this explanation is inverted.  The object that implements `===` defines the category to which the passed object may (or may not) belong, but as written, "declares whether or not it belongs in the same category as the other object" reads like the inverse.